test/addons-napi/test_reference/test_reference.c

```diff
@@ -61,7 +61,7 @@ napi_value CheckExternal(napi_env env, napi_callback_info info) {
   NAPI_ASSERT(env, argtype == napi_external, "Expected an external value.")
 
   int* data;
-  NAPI_CALL(env, napi_get_value_external(env, arg, &data));
+  NAPI_CALL(env, napi_get_value_external(env, arg, (void*)&data));
```

Comment:
This should be `void**`, not `void*`. Alternatively, you could also lose the pointer-to-pointer cast and do it the safe way:
```c
void* data;
napi_get_value_external(env, arg, &data);
data != NULL && *(int*)data == test_value
```